aerosonicdb/models/mfcc/cnn.py

```diff
@@ -117,12 +136,13 @@ def run_cv(train_path=TRAIN_PATH,
         ap_score = average_precision_score(y_true=y_test, y_score=y_prob)
         eval_results.append(ap_score)
 
-        PrecisionRecallDisplay.from_predictions(y_test, y_prob, ax=ax, name=f'CNN {count}')
+        PrecisionRecallDisplay.from_predictions(
+            y_test, y_prob, ax=ax, name=f"CNN {count}"
+        )
 
         if save_models:
-
             # save the model
-            model_path = os.path.join(output_path, f'cnn_{count}', 'model')
+            model_path = os.path.join(output_path, f"cnn_{count}", "model.keras")

```

Comment:
without the `.keras` extension this was crashing - keras now requires this extension apparently.